server/api.js

```diff
@@ -121,31 +122,81 @@ function listTitles(req, res, next) {
  *
  *
  */
-function REGISTER_USER(req, res, next) {
+
+// Todo: This function might well be obsolete
```

Comment:
don't redirect, if we don't know the user then simply return 403 like the below returns 401
```
  return function (req, res, next) {
    if (!req.user) {
      if (realm) res.set('WWW-Authenticate', `Bearer realm="${realm}"`);
      res.sendStatus(401);
    } else {
      next();
    }
  };
```